kytos/core/controller.py

```diff
@@ -629,7 +629,7 @@ def set_switch_options(self, dpid):
 
         vlan_pool = {}
         try:
-            vlan_pool = json.loads(self.options.vlan_pool)
+            vlan_pool = self.options.vlan_pool
             if not vlan_pool:
                 return
         except (TypeError, json.JSONDecodeError) as err:
```

Comment:
@slinksoft You've removed the call to `json.loads` but left the check for the exceptions that would be generated by it. I think this whole block can be replaced by just: `vlan_pool = self.options.vlan_pool`